ui/src/App.tsx

```diff
@@ -7,25 +7,27 @@ import Metrics from "./pages/Metrics/Metrics";
 
 import { DashboardInfo } from "./api/Api";
 
-function App(props: { dashboard: DashboardInfo }) {
+interface IAppProps {
```

Comment:
@driptanil move this to a interface file where you can store all type for this folder structure.

Reply:
![image](https://github.com/waterdipai/datachecks/assets/53353462/8435f4bc-0e92-4b6c-bf6a-fca50bad8832)
If I don't use interface I cannot make App as React.FunctionalComponent
What do I do then ?  I think using interface this works fine

Reply:
Please use interface but create or find a dedicated file to store it